inductiva/tests/tasks/test_run_task.py

```diff
@@ -0,0 +1,51 @@
+"""Test file for the run_simulation file."""
+import os
+import json
+
+import pathlib
+
+from inductiva import tasks
+
+LOGGED_TASK_ID = "1234"
+NOT_LOGGED_TASK_ID = "4321"
+
+
+def _id_in_metadata_file(task_id):
+    """Checks if the ID exists in the file."""
+    with open(tasks.TASK_METADATA_FILENAME, "r", encoding="utf-8") as f:
+        json_list = [json.loads(line) for line in f]
+    return task_id in map(lambda x: x["task_id"], json_list)
+
+
+def test_run_simulation_logging(mocker):
```

Comment:
Currently, we are trying to follow a more descriptive and focused pattern in the tests. 

For the descriptive part, the suggestion is to give a first name to the test that represents the function we are testing, then what we are inputing and then the expected output or setting you are using. 

For example, for this case it could be something like: `test_run_simulation__valid_task__write_in_metadata_file`. 

With this, your tests can be more focused on just the setting where the environment variable is not set. Then, you can create another one for the reverse. In this way, when a test fails, we will immediately know the case which is failing. 

To finish, don't be afraid to add a parameterization that tests more than one case (tests are the first QA barrier). For all of it, check the `mixins/test_filemanager.py`